scalanet/test/src/io/iohk/scalanet/peergroup/InMemoryPeerGroupSpec.scala

```diff
@@ -53,15 +55,15 @@ class InMemoryPeerGroupSpec extends FlatSpec with BeforeAndAfter {
       bobServer.futureValue.to shouldBe alice.processAddress
     }
 
-  it should "not receive a channel that started before the subscription to the server() stream" in
+  ignore should "not receive a channel that started before the subscription to the server() stream" in
```

Comment:
is this the test we noticed to be flaky?

Reply:
Yes.